### PR TITLE
Add intro to graphs quiz

### PR DESCRIPTION
**Pull request by nathangong**
Let me know if there are any other questions I should add.

### CODE REVIEW

**Comment by caoash** [comment]
Looks good, I have no issues with this. Ping me after a few content authors take a look and I'll merge.

File: content/2_Bronze/Intro_Graphs.mdx [hunk 1]
```diff
@@ -74,3 +75,180 @@ Knowing [DFS](/silver/dfs) can be helpful but it should not be required.
...
+      0 & 0 & 0 & 0 & 1 & 1\\
+      0 & 0 & 0 & 0 & 0 & 1\\
+      \end{pmatrix}
+      $
+      <Quiz.Explanation>
+        Incorrect. This is an adjacency matrix representation of the graph, not an adjacency list.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      0: [1, 5]
```
Comment (nwatx) [inline content/2_Bronze/Intro_Graphs.mdx:130]: could we texify this and the following one

**Review by nwatx — CHANGES_REQUESTED** [review]
other than this, well done!

**Review by SansPapyrus683 — APPROVED** [review]
alr yeah the questions are cool

**Comment by nathangong** [comment]
Does anyone know how to resize images in markdown? I added higher quality images but they ended up becoming really big

**Comment by nathangong** [comment]
Temporarily converting to a draft so i can resize the images

File: content/2_Bronze/Intro_Graphs.mdx [hunk 2]
```diff
@@ -74,3 +75,188 @@ Knowing [DFS](/silver/dfs) can be helpful but it should not be required.
 ## Problems
 
 <Problems problems="general" />
+
+## Check Your Understanding
+
+<Quiz>
+  <Quiz.Question>
+    How many connected components are in the following graph?
+		<div height="200">
```
Comment (nwatx) [inline content/2_Bronze/Intro_Graphs.mdx:84]: by the way this doesn't work. Use [tailwind heights][(https://tailwindcss.com/docs/height) as a class instead (using `className`)

Comment (nathangong) [reply content/2_Bronze/Intro_Graphs.mdx:84]: whoops yeah I think I forgot to remove that

Comment (nathangong) [reply content/2_Bronze/Intro_Graphs.mdx]: I ended up just manually decreasing the resolution of the images to decrease their size, but let me know if you know of a better way to do so

Comment (nwatx) [reply content/2_Bronze/Intro_Graphs.mdx]: use tailwind heights!

Comment (nathangong) [reply content/2_Bronze/Intro_Graphs.mdx]: it looks like this when i try it
image

**Review by nwatx — APPROVED** [review]

**Review by ryanchou-dev — APPROVED** [review]

File: content/2_Bronze/Intro_Graphs.mdx [hunk 3]
```diff
@@ -74,3 +75,180 @@ Knowing [DFS](/silver/dfs) can be helpful but it should not be required.
...
+    <Quiz.Answer>
+      2
+      <Quiz.Explanation>
+        
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      3
+      <Quiz.Explanation>
+        Correct. The connected components consist of $(0, 1), (2, 3, 4, 5), (6, 7, 8)$
```
Comment (maggie-j-liu) [inline content/2_Bronze/Intro_Graphs.mdx:100]: maybe use curly braces instead of parentheses?

**Review by maggie-j-liu — APPROVED** [review]

**Pull request merged**